calcs.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: calcs
+title: Arterial Management Division Reference Calculators
+title_short: Austin Transportation Department
+description: Austin Transportation Department
+custom_js:
+- calcs.bundle
+custom_css:
+- custom
```

Comment:
Can we put CSS in a separate file instead of adding it to the bottom of a very general file? This will help keep our more code organized and modular.

Reply:
Done!